docs/RELEASE_NOTES.md

```diff
@@ -7,6 +7,7 @@
   
 - Changes
   - fixed an issue where the operator would reconcile based on events emitted by itself in certain situations.
+  - expose connection strings (including auth/tls values) for deployments as secrets for easy of use. Secrets name template: _\<MongoDB resource name\>-\<db\>-\<user\>_
```

Comment:
Can we also add an entry about supporting SRV now too? Maybe just
`- support connection strings using SRV` 
or similar.

Reply:
sure, done